astropy/wcs/wcsapi/low_level_api.py

```diff
@@ -278,6 +278,26 @@ def serialized_classes(self):
         """
         return False
 
+    def _as_mpl_axes(self):
+        """
+        Compatibility hook for Matplotlib and WCSAxes. With this method, one can
+        do::
+
+            from astropy.wcs import WCS
+            import matplotlib.pyplot as plt
+            wcs = WCS('filename.fits')
+            fig = plt.figure()
+            ax = fig.add_axes([0.15, 0.1, 0.8, 0.8], projection=wcs)
+            ...
+
+        and this will generate a plot with the correct WCS coordinates on the
+        axes.
+        """
+
+        # Note this method taken from `astropy.wcs.WCS`
```

Comment:
Do we need this comment?!

Reply:
no and also I just realized we can actually move the equivalent method on WCS since it's inherited from here now